opacus/tests/scheduler_test.py

```diff
@@ -46,7 +46,9 @@ def test_step_scheduler(self):
         self.assertEqual(self.optimizer.noise_multiplier, gamma ** 2)
 
     def test_lambda_scheduler(self):
-        noise_lambda = lambda epoch: (1 - epoch / 10)
+        def noise_lambda(epoch):
+            return 1 - epoch / 10
```

Comment:
Btw, why not use lambda syntax here?

I would even put it right in place where it is used (sorry, I'm allergic to spaghetti code):
```python
scheduler = LambdaNoise(
    self.optimizer,
    noise_lambda=lambda epoch: 1 - epoch / 10,
)
```

Reply:
I was surprised too
https://www.flake8rules.com/rules/E731.html

Reply:
Right, but if we use lambda inline to pass the argument (as in my example), it's not a violation of E731.